include/revng/Model/StructDefinition.h

```diff
@@ -28,21 +28,37 @@ TUPLE-TREE-YAML */
 #include "revng/Model/Generated/Early/StructDefinition.h"
 
 class model::StructDefinition : public model::generated::StructDefinition {
-public:
-  static constexpr const char *AutomaticNamePrefix = "struct_";
-
 public:
   using generated::StructDefinition::StructDefinition;
+  explicit StructDefinition(uint64_t StructSize) :
+    generated::StructDefinition() {
 
-public:
-  Identifier name() const;
+    Size() = StructSize;
+  }
+
+  StructField &addField(uint64_t Offset, UpcastableType &&Type) {
```

Comment:
So you can do `.addField(...).addField(...)`?
:galaxybrain:

Reply:
I picked that up from Jack with his `PTMLBuilder` I think :enterprise: